python/sglang/srt/layers/attention/trtllm_mla_backend.py

```diff
@@ -474,3 +480,20 @@ def forward_decode(
         output = raw_out_v.view(-1, layer.tp_q_head_num * layer.v_head_dim)
 
         return output
+
+
+class TRTLLMMLAMultiStepDraftBackend(FlashInferMLAMultiStepDraftBackend):
+    """Multi-step draft backend for TRT-LLM MLA used by EAGLE."""
+
+    def __init__(
+        self, model_runner: "ModelRunner", topk: int, speculative_num_steps: int
+    ):
+        super().__init__(model_runner, topk, speculative_num_steps)
```

Comment:
Flashinfer MLA Backend restricts on topk=1. Can we ensure the accuracy for topk=1 and hard errors when topk != 1?